common/source/docs/common-optional-hardware.rst

```diff
@@ -76,6 +76,7 @@ information related to Autopilot selection see :ref:`Autopilot Hardware Options
 [/site]
     Telemetry Radio <common-telemetry-landingpage>
     UAVCAN Adapter Node <common-uavcan-adapter-node>
+    UAVCAN Perpherals <common-uavcan-peripherals>
```

Comment:
missing the "i" .. should be "Peripherals"

Reply:
fixed